<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1568258509331" stop="1568258509452">
  <name>test01</name>
  <labels/>
  <test-cases>
    <test-case start="1568258509331" status="passed" stop="1568258509356">
      <name>Test.test001</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3092-MainThread"/>
        <label name="host" value="LAPTOP-MME9N47P"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1568258509365" status="passed" stop="1568258509367">
      <name>Test.test002</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3092-MainThread"/>
        <label name="host" value="LAPTOP-MME9N47P"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1568258509368" status="failed" stop="1568258509452">
      <name>Test.test003</name>
      <failure>
        <message>AssertionError: assert 0</message>
        <stack-trace>self = &lt;test01.Test object at 0x03A9D050&gt;

    def test003(self):
        print("test003被执行")
        # 断言失败
&gt;       assert 0
E       assert 0

scripts\test01.py:11: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="3092-MainThread"/>
        <label name="host" value="LAPTOP-MME9N47P"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
